build-system/tasks/css.js

```diff
@@ -54,6 +54,11 @@ const cssEntryPoints = [
     // than the JS file to avoid loading CSS as JS
     outCss: 'video-autoplay-out.css',
   },
+  {
+    path: 'amp-story-player.css',
+    outJs: 'amp-story-player.css.js',
+    outCss: 'amp-story-player-out.css',
```

Comment:
What's the `outJs` for? Can we rename the `outCss` to have something more publisher friendly?